app/models/web_hook.rb

```diff
@@ -57,18 +61,16 @@ def self.enqueue_topic_hooks(event, topic, user)
   %i(post_created
      post_destroyed
      post_recovered).each do |event|
-
     DiscourseEvent.on(event) do |post, _, user|
-      WebHook.enqueue_hooks(:post,
-        post_id: post.id,
-        topic_id: post&.topic&.id,
-        user_id: user&.id,
-        category_id: post.topic&.category&.id,
-        event_name: event.to_s
-      )
+      WebHook.enqueue_post_hooks(event, post, user)
     end
   end
 
+  DiscourseEvent.on(:before_edit_post) do |post, topic_changed|
+    WebHook.enqueue_post_hooks(:before_edit_post, post)
```

Comment:
I was worried they may interested in where the event initially happens. But it seems they just need to distinguish them..so goes for `x_edited`